packages/docusaurus/README.md

```diff
@@ -1 +1,74 @@
+
+## Installation
+
+We have released the [`@docusaurus/init` package](https://www.npmjs.com/package/@docusaurus/init) to make [getting started](https://v2.docusaurus.io/docs/) with Docusaurus easy.
+
+## Contributing
+
+We've released Docusaurus because it helps us better scale and supports the many OSS projects at Facebook. We hope that other organizations can benefit from the project. We are thankful for any contributions from the community.
+
+### [Code of Conduct](https://code.fb.com/codeofconduct)
+
+Facebook has adopted a Code of Conduct that we expect project participants to adhere to. Please read [the full text](https://code.fb.com/codeofconduct) so that you can understand what actions will and will not be tolerated.
+
+## Contact
+
+We have a few channels for contact:
+
+- [Discord](https://discord.gg/docusaurus) with two text channels:
+  - `#docusaurus-users` for those using Docusaurus.
+  - `#docusaurus-dev` for those wanting to contribute to the Docusaurus core.
+- [@docusaurus](https://twitter.com/docusaurus) on Twitter
+- [GitHub Issues](https://github.com/facebook/docusaurus/issues)
+
+## Contributors
+
+This project exists thanks to all the people who contribute. [[Contribute](CONTRIBUTING.md)]. <a href="https://github.com/facebook/docusaurus/graphs/contributors"><img src="https://opencollective.com/Docusaurus/contributors.svg?width=890&button=false" /></a>
+
+## Backers
+
+Thank you to all our backers! 🙏 [Become a backer](https://opencollective.com/Docusaurus#backer)
+
+<a href="https://opencollective.com/Docusaurus#backers" target="_blank"><img src="https://opencollective.com/Docusaurus/backers.svg?width=890"></a>
+
+## Sponsors
+
+Support this project by becoming a sponsor. Your logo will show up here with a link to your website. [Become a sponsor](https://opencollective.com/Docusaurus#sponsor)
+
+<a href="https://opencollective.com/Docusaurus/sponsor/0/website" target="_blank"><img src="https://opencollective.com/Docusaurus/sponsor/0/avatar.svg"></a> <a href="https://opencollective.com/Docusaurus/sponsor/1/website" target="_blank"><img src="https://opencollective.com/Docusaurus/sponsor/1/avatar.svg"></a>
+
+## License
+
+Docusaurus is [MIT licensed](./LICENSE).
+
+The Docusaurus documentation (e.g., `.md` files in the `/docs` folder) is [Creative Commons licensed](./LICENSE-docs).
+
+## Special thanks
+
+[![BrowserStack logo](./admin/img/browserstack-logo.png)](http://www.browserstack.com/)
+
+[BrowserStack](http://www.browserstack.com/) supports us with [free access for open source](https://www.browserstack.com/open-source).
+
+[![Rocket Validator logo](./admin/img/rocketvalidator-logo.png)](https://rocketvalidator.com/)
+
+[Rocket Validator](https://rocketvalidator.com/) helps us find HTML markup or accessibility issues.
```

Comment:
```suggestion

```
Lets keep it empty for now as it's a duplicate of the homepage and many links are to v1, we'll need to update this soon anyway